src/sql/src/session/vars/definitions.rs

```diff
@@ -1545,6 +1545,16 @@ pub mod cluster_scheduling {
         "Always provisions a replica with disk, regardless of `DISK` DDL option.",
         true,
     );
+
+    const DEFAULT_CHECK_SCHEDULING_POLICIES_INTERVAL: u64 = 3;
```

Comment:
Not really. For users, it would be ok to go up to 10 sec, I think. I just didn't want the tests to take too long, but I guess I could just change the variable to a low value in the tests.

We can tune this after deciding what exactly to put on background tasks.

Reply:
I've added metrics for how long `check_refresh_policy` and `handle_scheduling_decisions` take, and it's almost always inside 1ms, even with 10 clusters and 20 MVs, when I tested it locally. (For `check_refresh_policy`, I only measured that part of it which runs on the main thread, i.e., querying the timestamp oracle is not included.)

So, I guess a 3 sec interval is fine.